kie-dmn-feel/src/main/java/org/kie/dmn/feel/lang/Type.java

```diff
@@ -24,4 +42,36 @@
 
     String toString( Object value );
 
+    public static Type determineTypeFromClass( Class<?> clazz ) {
```

Comment:
I am not a big fan of mixing behaviour with interfaces. I think it would be better to have this method on a separate class to keep the interface clean.